src/rust/dbv2/migrations/2023-08-25-111710_add_market_registration_events/up.sql

```diff
@@ -43,6 +43,39 @@ AFTER INSERT ON market_registration_events FOR EACH ROW
 EXECUTE PROCEDURE notify_market_registration_event ();
 
 
+CREATE TABLE
+  fill_events (
+    txn_version NUMERIC(20) NOT NULL,
+    event_idx NUMERIC(20) NOT NULL,
+    PRIMARY KEY (txn_version, event_idx),
+    emit_address VARCHAR(70) NOT NULL,
+    time timestamptz NOT NULL,
+    maker_address VARCHAR(70) NOT NULL,
+    maker_custodian_id NUMERIC(20) NOT NULL,
+    maker_order_id NUMERIC(40) NOT NULL,
+    maker_side BOOLEAN NOT NULL,
+    market_id NUMERIC(20) NOT NULL,
+    price NUMERIC(20) NOT NULL,
+    trade_sequence_number NUMERIC(20) NOT NULL,
+    size NUMERIC(20) NOT NULL,
+    taker_address VARCHAR(70) NOT NULL,
+    taker_custodian_id NUMERIC(20) NOT NULL,
+    taker_order_id NUMERIC(40) NOT NULL,
+    taker_quote_fees_paid NUMERIC(20) NOT NULL
+  );
+
+CREATE FUNCTION notify_fill_event () RETURNS TRIGGER AS $$
+BEGIN
+   PERFORM pg_notify('fill_event'::text, row_to_json(NEW)::text);
+   RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+
+
+CREATE TRIGGER fill_events_trigger
+AFTER INSERT ON fill_events FOR EACH ROW
+EXECUTE PROCEDURE notify_fill_event ();
+
```

Comment:
I would have done this in a new migration. This one is called `market_registration_events`.

Reply:
That's true, I was having trouble doing it in a second migration so did it here. Can try again.